test/videojs.ads.test.js

```diff
@@ -709,17 +708,14 @@ test('adscanceled allows us to transition from content-set to content-playback',
 
 test('adscanceled allows us to transition from ads-ready? to content-playback', function() {
   var callback;
-  expect(6);
+  expect(5);
```

Comment:
@dmlap @gkatsev Upon further review, I removed the previous test updates I had made alltogether.  You will notice that if you run the tests on master in a browser now they will throw with this error:

```
Uncaught Error: assertion outside test context, was     at window.clearImmediate (http://localhost:8081/test/videojs.ads.test.js:720:21)
    at clearImmediate (http://localhost:8081/src/videojs.ads.js:104:33)
    at Object.fsm.preroll?.leave (http://localhost:8081/src/videojs.ads.js:363:15)
    at http://localhost:8081/src/videojs.ads.js:477:38
    at fsmHandler (http://localhost:8081/src/videojs.ads.js:485:9)
    at HTMLDivElement.e (http://localhost:8081/node_modules/video.js/dist/video-js/video.js:13:254)
    at HTMLDivElement.e.Y.e.Y (http://localhost:8081/node_modules/video.js/dist/video-js/video.js:6:286)
Source: 	
http://localhost:8081/libs/qunit/qunit.js:899
```
 The root cause of this was that clearTimeout was not restored in the AdFramework teardown even though its overridden in the tests.  My last commit fixes this and should make this pull a bit more tolerable.